project/Sonatype.scala

```diff
@@ -0,0 +1,71 @@
+
+/*
+ Copyright 2015 Coursera Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+package org.coursera.naptime.sbt
```

Comment:
[We don't use a package in coursera/courscala](https://github.com/coursera/courscala/blob/aeb45be956abd744eb49a59113595470a8d83594/project/Sonatype.scala); do we need one here?